File: ChibuzorTasks/FindLargestAndSmallest.java
import java.util.Scanner;

public class FindLargestAndSmallest {
	public static void main(String[] args){
		Scanner input = new Scanner(System.in);
		int largest = 0;
		int smallest = 0;
		String repeat;

		System.out.print("Enter your First Number: ");
		int firstNumber = input.nextInt();

		System.out.print("Enter your Second Number: ");
		int secondNumber = input.nextInt();

		if(firstNumber > secondNumber) {
			largest = firstNumber;
			smallest = secondNumber;
		} else {
			largest = secondNumber;
			smallest = firstNumber;
		}
		
		do {
			input.nextLine();
			System.out.println("Would you like to enter another number (yes/no) ");
			repeat = input.nextLine();
		
		
			if (repeat.equals("yes")) {
		
				System.out.print("Enter your Another Number: ");
				int anotherNumber = input.nextInt();

				if(anotherNumber > firstNumber ) {
					largest = anotherNumber;

				} else if(anotherNumber < secondNumber) {
					smallest = anotherNumber;
				}
			}
		} while (repeat.equals("yes"));
	
		System.out.println("The Largest number is : " + largest);
		System.out.println("The Smallest number is : " + smallest);
		

	}


}

File: ChibuzorTasks/find_largest_and_smallest.py
largest = 0
smallest = 0
repeat =""

firstNumber = int(input("Enter your First Number: "))
secondNumber = int(input("Enter your second Number: "))	

if(firstNumber > secondNumber):
	largest = firstNumber
	smallest = secondNumber
else :
	largest = secondNumber
	smallest = firstNumber

		
while True:
	repeat = input("Would you like to enter another number (yes/no): ")
	if (repeat != "yes"):
		break 
	
	anotherNumber = int(input("Enter your Another Number: "))
	if(anotherNumber > firstNumber ): 
		largest = anotherNumber
	elif(anotherNumber < secondNumber):
		smallest = anotherNumber
					
print(f"The Largest number is : {largest}")
print(f"The Smallest number is : {smallest}")